1) Когда используется критерий Стьюдента, а когда Z –критерий?

Ответ: Критерий Стьюдента используется, когда сигма генеральной совокупности неизвестна, соблюдается нормальность, а также соблюдается равенство дисперсий в сравниваемых группах. Z-критерий используется, когда сигма генеральной совокупности известна, а также соблюдается нормальность.

2. Проведите тест гипотезы. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.

Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр

оказался равным 17.5 мм, а дисперсия известна и равна 4 кв. мм. (Вручную)

In [2]:
import math

In [3]:
a = 0.05
n = 100
disp = 4
sigma = math.sqrt(disp)
mu0 = 17
mu1 = 17.5

H0: mu = mu0

H1: mu > mu0

Так как сигма генеральной совокупности известна => используем Z-критерий.

In [4]:
zn = (mu1 - mu0)/(sigma / math.sqrt(n))
zn

2.5

In [5]:
zt = 1.65

In [6]:
zn > zt

True

Гипотеза Н1 верна.

3. Проведите тест гипотезы. Продавец утверждает, что средний вес пачки печенья составляет 200 г.

Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:

202, 203, 199, 197, 195, 201, 200, 204, 194, 190.

Известно, что их веса распределены нормально.

Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%? (Провести двусторонний тест.)

In [7]:
import numpy as np

In [37]:
n = 10
a = 0.01
m = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])
mu0 = 200

In [24]:
mu1 = np.mean(m)
mu1

198.5

In [33]:
s2 = sum((i - mu1)**2 for i in m)/(n - 1)
s2

19.833333333333332

In [36]:
s = math.sqrt(s2)
s


4.453463071962462

H0: mu = 200

H1: mu != 200

Так как сигма генеральной совокупности неизвестна => используем критерий Стьюдента.


In [38]:
tn = (mu1 - mu0) / (s / np.sqrt(n))
tn

-1.0651074037450896

In [40]:
tt = 3.25

In [41]:
np.abs(tn) < tt

True

Следовательно, нулевая гипотеза верна.

4. Есть ли статистически значимые различия в росте дочерей?

Рост матерей 172, 177, 158, 170, 178,175, 164, 160, 169

Рост взрослых дочерей: 173, 175, 162, 174, 175, 168, 155, 170, 160

In [42]:
m = np.array([172, 177, 158, 170, 178,175, 164, 160, 169])
d = np.array([173, 175, 162, 174, 175, 168, 155, 170, 160])

H1: mu = mu1

H2: mu != mu1

In [44]:
stats.ttest_rel(m, d)

Ttest_relResult(statistic=0.559522990335608, pvalue=0.5911212354055175)

Так как pvalue = 0.59 и больше альфа = 0.05 => принимаем нулевую гипотезу